app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/app/tracking/NetpRealAppTrackerDetector.kt

```diff
@@ -64,7 +64,7 @@ class NetpRealAppTrackerDetector constructor(
         val packageId = appNameResolver.getPackageIdForUid(uid) ?: return null
 
         if (VpnExclusionList.isDdgApp(packageId) || packageId.isInExclusionList()) {
-            logcat { "shouldAllowDomain: $packageId is excluded, allowing $domain" }
+            logcat { "shouldAllowDomain: $packageId is excluded, allowing packet" }
```

Comment:
Why the change?

Reply:
When I was writing up privacy triage I realized that if our internal users exclude an app, they may not want the app's domains to show up in the logs. Especially since we sometimes ask our internal users to send us logs for extra debugging.

Reply:
Not sure I agree with that. Those logs are primarily for us dev in debug mode.
If internal users don't want to send the logs they just don't. Also, these logs are not enabled in prod or internal builds. 

Not sure what privacy triage has to do here tbf.

Reply:
We had an offline conversation about privacy triage being needed or not. Regardless, the triage got approved and I would prefer to keep this change if you don't mind.